src/constants/environment-vars.constants.ts

```diff
@@ -1,5 +1,7 @@
-import * as dotenv from 'dotenv';
+import * as dotenv from "dotenv";
```

Comment:
Single quotes not good enough for you? 😂 

If you're going to change them here make sure you change them everywhere.